### DISCUSSION TITLE
Import from mobilizon.evns.net ?

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#399] [Ideas]
I just created todo issues to add several venues in Berlin, and then I discovered that they are already there in this Mobilizòn instance.

I could very easily write a script which could convert from the Mobilizòn API to the concert-cloud API. I've done it the other way around of course, but this would be a bit easier I think because the concert-cloud API is a lot less demanding.

**Comment by MarkJaroski** [comment]
Well, maybe not ... I realized that all of those events are federated from https://mobilize.berlin, which is great, but they seem to be hand-entered and incomplete.

I suppose that scraping would still be better.

**Reply by jakopako** [reply]
I see, yeah I guess if they're incomplete, scraping makes more sense.
